src/main/java/duke/Task.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String category;
+    protected String at;
+    protected String by;
+
+    public Task(String description) {
+        this.description = description;
+        category = "";
+        isDone = false;
+        at = "";
+        by = "";
+    }
+
+    //Get a icon for the task
+    public String getStatusIcon() {
+        if(!isDone) {
+            return ("\u2718");
```

Comment:
Will this be a potential bug? The tick and cross icon not being shown correctly.

Reply:
Hi Justin, I dont get this part. What you mean by potential bug ?